cadence/lib/js/flow.json

```diff
@@ -0,0 +1,21 @@
+{
```

Comment:
It would be good to use the `flow.json` in the root of the repo if possible.

Reply:
@MaxStalker Note: we'll create an issue in `flow-js-testing` for this

Reply:
https://github.com/onflow/flow-js-testing/issues/32